23homework.sql

```diff
@@ -0,0 +1,108 @@
+-- Tworzy tabelę pracownik(imie, nazwisko, wyplata, data urodzenia, stanowisko). W tabeli mogą być dodatkowe kolumny, które uznasz za niezbędne.
+CREATE TABLE pracownicy (
+	id_pracownika int primary key auto_increment not null,
+    imie varchar(20) NOT NULL,
+    nazwisko varchar(50) NOT NULL,
+    wypłata int,
+	data_urodzenia date NOT NULL,
+    stanowisko varchar(100) NOT NULL
+);
+
+-- Wstawia do tabeli co najmniej 6 pracowników
+INSERT INTO pracownicy(imie, nazwisko, wypłata, data_urodzenia, stanowisko) 
+VALUES ('Adam', 'Adamski', 5000, '1980-07-01', 'kierownik'), 
+('Ania', 'Kowalska', 1000, '1980-02-22', 'dyrektor'),
+('Krzyś', 'Pałka', 4000, '1980-09-01', 'specjalista'),
+('Robert', 'Jak', 3000, '1980-03-07', 'młodszy specjalista'),
+('Janina', 'Kowalczyk', 7000, '1980-10-15', 'administrator it'),
+('Oliwia', 'Nowak', 0, '2000-01-01', 'praktykant');
+
+-- Pobiera wszystkich pracowników i wyświetla ich w kolejności alfabetycznej po nazwisku
+select * from pracownicy ORDER BY nazwisko ASC;
+
+-- Pobiera pracowników na wybranym stanowisku
+select * from pracownicy where stanowisko ='dyrektor';
+
+-- Pobiera pracowników, którzy mają co najmniej 30 lat
+SELECT * FROM pracownicy
+WHERE TIMESTAMPDIFF(YEAR, data_urodzenia, CURDATE()) >= 30;
+
+-- Zwiększa wypłatę pracowników na wybranym stanowisku o 10%
+select imie, nazwisko, wypłata*1.1 as wypłata from pracownicy;
```

Comment:
To nie zwieksza wyplaty, jedynie wyswietla wyplate o 1.1 raza wyzsza, potrzebujesz wykorzystac update. Dodatkowo zrob to dla tylko jednego stanowiska ;)